plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/src/com/google/cloud/tools/eclipse/appengine/deploy/ui/StandardDeployPreferencesPanel.java

```diff
@@ -286,15 +280,18 @@ private void createCredentialSection(IGoogleLoginService loginService) {
     // If we don't require values, then don't auto-select accounts
     accountSelector = new AccountSelector(accountComposite, loginService,
         Messages.getString("deploy.preferences.dialog.accountSelector.login"), requireValues);
-    GridLayoutFactory.fillDefaults().numColumns(2).generateLayout(accountComposite);
+   GridLayoutFactory.fillDefaults().numColumns(2).generateLayout(accountComposite);
   }
 
   private void createProjectIdSection() {
     Composite projectIdComposite = new Composite(this, SWT.NONE);
 
     projectIdLabel = new Label(projectIdComposite, SWT.LEFT);
     projectIdLabel.setText(Messages.getString("project.id"));
-
+    GridData layoutData = GridDataFactory.swtDefaults().create();
+    layoutData.horizontalAlignment = GridData.BEGINNING;
```

Comment:
Not an export of SWT, but the Javadoc of `swtDefaults()` says `BEGINNING` is the default. Still need this?

```
     * <p>
     * Initial values are:
     * </p>
     *
     * <ul>
     * <li>align(SWT.BEGINNING, SWT.CENTER)</li>
     * <li>exclude(false)</li>
     * <li>grab(false, false)</li>
     * <li>hint(SWT.DEFAULT, SWT.DEFAULT)</li>
     * <li>indent(0,0)</li>
     * <li>minSize(0,0)</li>
     * <li>span(1,1)</li>
     * </ul>
```